README.md

```diff
@@ -19,16 +19,28 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 ## Preprocessing
 
-*What preprocessing data do you store in files, and how?*
+In the preprocessing file we have 3 types of entries. The first entry marked by '>' is the name of the genome. The second entry is marked by '@' and it is the last column in the burrow-wheeler matrix, bwt. After this we write each entry from the C array, which is the accumulated buckets of the suffix array. Now each new entry is written on a new line in the file and is marked by '\*', where the first symbol following '\*' marks the symbol in the suffix array this is followed by an integer value indicating the starting position of the symbol in the suffix array.
+We do not create the O table as it would still require order n time to reconstruct and it is roughly as fast to build directly from the bwt.
+
 
 ## Insights you may have had while implementing the algorithm
 
+We never really considered that preprocessing and by saving some intermediate result to a file for later use could be of benefit and faster use when read-mapping or for any other algorithms for that matter.
```

Comment:
Then I am happy that I had you make this project. It is an insanely useful idea.